src/test/java/core/basesyntax/RegistrationValidationTests.java

```diff
@@ -0,0 +1,56 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.Test;
+
+public class RegistrationValidationTests {
+
+    private User validUser = new User(2556465L, "someuser", "userpass", 56);
+    private User usersameAsValidUser = new User(2556465L, "someuser", "userpass", 56);
+    private User userWrongPass = new User(25458965L, "someuser1", "us1pa", 25);
+    private User userWrongAge = new User(25234465L, "someuser3", "user3password", 17);
+    private User userWrongLogin = new User(25234465L, "user3", "user5password", 25);
+
+    @Test
+    public void duplicate_user_registration_NotOk() {
```

Comment:
```suggestion
    public void register_duplicate_user_registration_NotOk() {
```
follow pattern from task description